packages/data-context/test/unit/actions/MigrationActions.spec.ts

```diff
@@ -91,7 +90,6 @@ describe('MigrationActions', () => {
       ctx.update((coreData) => {
         coreData.currentProject = currentProject
         coreData.currentTestingType = 'e2e'
-        coreData.app.isInGlobalMode = true
       })
```

Comment:
This was some very sticky logic! Before this PR, `isInGlobalMode` really meant `currentProject is falsy`. In `packages/graphql/src/schemaTypes/objectTypes/gql-Query.ts` we had the following logic:
```
t.nonNull.boolean('isGlobalMode', {
      description: 'Whether the app is in global mode or not',
      resolve: (source, args, ctx) => !ctx.currentProject,
    })
```
As a result, some data massaging was necessary to keep `isInGlobalMode` true in `MigrationActions.spec.ts` when the currentProject was created.

Now, this logic has been removed and `isGlobalMode` refers ONLY to whether the app was opened with the flag `--global` per the `modeOptions` object